core/ui/mpc/keysign/KeysignSigningStep.tsx

```
@@ -37,123 +46,218 @@ export const KeysignSigningStep = ({ onBack }: KeysignSigningStepProps) => {
+        setBlockaidWarning(error.blockaid || null)
+        setShowWarningModal(true)
+        return
+      }
+    }
+    if (mutationStatus.status !== 'success') return
+    const txResults = mutationStatus.data
+    if (!txResults || !Array.isArray(txResults) || txResults.length === 0)
+      return
+    if (!('keysign' in payload)) return
+    const keysignPayload = payload.keysign
+    if (keysignPayload.swapPayload && keysignPayload.swapPayload.value) return
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)